packages/create-neutrino-project/commands/init/index.js

```diff
@@ -0,0 +1,187 @@
+const { ensureDirSync, readJsonSync, writeJsonSync } = require('fs-extra');
+const path = require('path');
+const chalk = require('chalk');
+const stringify = require('javascript-stringify');
+const Generator = require('yeoman-generator');
+const questions = require('./questions');
+const {
+  PROJECTS,
+  LIBRARIES,
+  LINTERS,
+  isYarn,
+  isObjectEmpty
+} = require('./utils');
+
+module.exports = class Project extends Generator {
+  _makeRcFile(data) {
+    let config = { use: [] };
```

Comment:
This can be `const` since you aren't overwriting the value of `config`.

Reply:
++